packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -946,6 +946,22 @@ class EditableText extends StatefulWidget {
   /// {@end-tool}
   /// {@endtemplate}
   ///
+  /// ## Handling emojis and other complex characters
+  /// {@template flutter.widgets.editableText.complexCharacters}
+  /// It's important to always use
```

Comment:
I think that's probably not true If the editable text is only meant for english alphabets and/or numbers (verification code, for example), and there's a formatter that guarantees that?

Reply:
True.  I'll change it to something like "when working with user input text that may contain complex characters"